meilisearch-http/src/option.rs

```diff
@@ -115,6 +115,18 @@ pub struct Opt {
     /// Defines time interval, in seconds, between each snapshot creation.
     #[structopt(long, requires = "snapshot-path", env = "MEILI_SNAPSHOT_INTERVAL_SEC")]
     pub snapshot_interval_sec: Option<u64>,
+
+    /// Path where backup will be created if backup route is called.
+    #[structopt(long, env = "MEILI_BACKUP_PATH", default_value = "backups/")]
+    pub backup_path: String,
+
+    /// Import a backup from the specified path, must be a `.tar.gz` file.
+    #[structopt(long, env = "MEILI_IMPORT_BACKUP", conflicts_with = "load-from-snapshot")]
+    pub import_backup: Option<PathBuf>,
```

Comment:
Here we have a `String` for the `backup_path` and an `Option<PathBuf>` for the `import_backup` option.

I would prefer to have a `PathBuf` and that the `backup_path` is renamed `backup_folder`, what do you think?